### PR TITLE
fix error urls

### PR DESCRIPTION
**Pull request by Aresforchina**
fix some 404 urls
ref:#17649
@Rajakavitha1

### CODE REVIEW

File: content/en/blog/_posts/2016-12-00-Kubernetes-1-5-Supporting-Production-Workloads.md [hunk 1]
```diff
@@ -58,9 +58,11 @@ Offline, the community can be found at one of the many Kubernetes related [meetu
 
 
 
+
 Ready to start contributing? Share your voice at our weekly [community meeting](https://kubernetes.io/community/).&nbsp;  
 
 
+
 - Post questions (or answer questions) on [Stack Overflow](http://stackoverflow.com/questions/tagged/kubernetes)&nbsp;
```
Comment (sftim) [inline content/en/blog/_posts/2016-12-00-Kubernetes-1-5-Supporting-Production-Workloads.md]: I recommend making no changes to this file; it's good as-is.

Comment (Aresforchina) [reply content/en/blog/_posts/2016-12-00-Kubernetes-1-5-Supporting-Production-Workloads.md]: @sftim I have deleted unused null line.

**Comment by zacharysarah** [comment]
@aresforchina In the future, check the [PR wrangler](https://github.com/kubernetes/website/wiki/PR-Wranglers) list and assign a reviewer once per week maximum. Please don't spam numerous reviewers over several days.

**Pull request merged**